pkg/ingress/controller/apisix_route.go

```diff
@@ -100,14 +100,15 @@ func (c *ApisixRouteController) updateFunc(oldObj, newObj interface{}) {
 }
 
 func (c *ApisixRouteController) deleteFunc(obj interface{}) {
+	oldRoute := obj.(cache.DeletedFinalStateUnknown).Obj.(*api6V1.ApisixRoute)
```

Comment:
Reference to istio: https://github.com/istio/istio/blob/18b3d35de0523862a389553b50b13ee4e73f4530/pilot/pkg/serviceregistry/kube/controller/namespacecontroller.go#L160

Reply:
Done